src/cli/README_HISTORY.md

```diff
@@ -459,11 +459,11 @@ Print the IPv6 message RX history as a table:
 |         00:00:30.000 | src: [fe80:0:0:0:d03d:d3e7:cc5e:7cd7]:19788                                 |
 |                      | dst: [fe80:0:0:0:a4a5:bbac:a8e:bd07]:19788                                  |
 +----------------------+------------------+-------+--------+-----+------+------+----+--------+-------+
-|                      | UDP              |    93 | 0x52df |  no |  net |  -20 | RX | unknwn |  15.4 |
+|                      | UDP              |    93 | 0x52df |  no |  net |  -20 | RX | unknown|  15.4 |
```

Comment:
This is not a typo (we use shorter string to keep table column width smaller).

I think we should not check the `cli` output for typo.

Reply:
I have added the word `unknwn` to the config file of the code spell. Then the word `unknwn` will not be considered a wrong word anymore.